backend/typescript/graphql/resolvers/authResolvers.ts

```diff
@@ -19,26 +22,60 @@ const cookieOptions: CookieOptions = {
   secure: process.env.NODE_ENV === "production",
 };
 
+// Object to pass back when frontend queries a login request
+class LoginOK {
```

Comment:
Can you look over these lint errors and fix all the ones that are reasonable to fix?

(there's a few things like it being upset about `console.error` that we can probably ignore)